gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -2364,14 +2373,58 @@ else if (value instanceof Traversal)
      * @since 3.0.0-incubating
      */
     public default GraphTraversal<S, E> property(final Object key, final Object value, final Object... keyValues) {
```

Comment:
haven't these changes now created a situation where you can't do: `property('data', [k:100])` with the expectation that you would store a `Map` at "data"? I don't think we want that to happen. right?

Reply:
Good catch, yes that is an error.  Fixed.